### PR TITLE
Front part one

### PR DESCRIPTION
**Pull request by miguelriverasarmiento**
Beginning of the front with the title.

### CODE REVIEW

File: src/public/css/styles.css [hunk 1]
```diff
@@ -1,3 +0,0 @@
-body{
```
Comment (jjmedinas) [inline src/public/css/styles.css:1]: Why did you delete these files?

File: src/views/index.html [hunk 2]
```diff
@@ -4,10 +4,13 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Ejs template</title>
+    <link rel="stylesheet" href="./css/styles.css" />
 </head>
 <body>
     
-    <h1>Ejs</h1>
+    <div id="titlebox">
```
Comment (jjmedinas) [inline src/views/index.html:11]: I think it's a good practice to use `class="titlebox"` instead of id=..., what do you think?
Is there any standard for this?

**Review by jjmedinas — APPROVED** [review]
Overall it looks good to me.  Good work!!
I left two comments just to understand why did make some changes (It doesn't mean it's wrong)

Please remember to follow the standard for commit messages: (Infinitive verb + complement)
e.g. Add frontend setup

File: src/public/css/styles.css (hunk 1 above)
Comment (miguelriverasarmiento) [reply src/public/css/styles.css:1]: The css file is in all commits, but i dont know here

File: src/views/index.html (hunk 2 above)
Comment (miguelriverasarmiento) [reply src/views/index.html:11]: I didn't know about that but I'm going to put it into practice
